environment.yml

```diff
@@ -4,10 +4,17 @@ channels:
   - nodefaults
 dependencies:
   - cupy~=13.3.0
+  - dask-jobqueue~=0.9.0
+  - rapidsai-nightly::kvikio>=25.04.00a
   - jupyterlab~=4.3.5
   - nvidia-dali-python~=1.45.0
-  - python=3.11
+  - nvtx~=0.2.11
```

Comment:
Am curious about the context for the `nvtx` version constraint

Reply:
Probably ok to unpin, I just applied a `~=` constraint for consistency in this environment.yml file